DRODLib/CharacterCommand.h

```diff
@@ -184,6 +184,17 @@ namespace ScriptFlag
 		Stalwart=4
 	};
 
+	enum GotoSmartType
+	{
+		LastIf = 0,
```

Comment:
I'd refer to "Last*" as "Previous*" to avoid ambiguity in whether this means jumping to the last (as in "final") If in the script.

Reply:
Good catch. This definitely a thing where I know what context "Last" is being used in that might be not be clear to a user.